stsynphot/commissioning/utils.py

```diff
@@ -3,7 +3,7 @@
 # STDLIB
 import os
 import sys
-from collections import Iterable
+from collections.abc import Sized
```

Comment:
Why the change here?
```suggestion
from collections.abc import Iterable
```

Reply:
from what I can see, this is asking for the `len()` of the object:
https://github.com/spacetelescope/stsynphot_refactor/blob/7c007f9caac36938df5581d9bac2bf4f6b828b38/stsynphot/commissioning/utils.py#L151-L154
so an `Iterable`, which only has to implement `__iter__()` may not implement a `__len__()` method. 
`Sized`, however, requires `__len__()` so would work in all cases.

Reply:
OK, that makes sense, I guess. And I don't think it matters in this case anyway, so I'll accept this. Thanks for the clarification!